src/app/ResetPasswordPage/ResetPasswordPage.jsx

```diff
@@ -1,30 +1,80 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useRouter } from 'next/router';
 import styled from 'styled-components';
 import ModalPage from '../../components/ModalPage';
 import Button from '../../components/Button';
 import FormItem from '../../components/FormItem';
 import Input from '../../components/Input';
 import Box from '../../components/Box';
+import resetPassword from '../../apis/auth/resetPassword';
+import config from './formConfig';
+import useForm from '../../hooks/useForm';
 
 const ConfirmButton = styled(Button)`
   margin-top: 32px;
 `;
 
-const ResetPasswordPage = () => (
-  <ModalPage title="重置密码">
-    <Box mt="md">您正在进行重制密码操作，请输入新密码。</Box>
-    <Box py="lg">
-      <FormItem layout="inline" label="密码">
-        <Input />
-      </FormItem>
-      <FormItem layout="inline" label="确认密码">
-        <Input />
-      </FormItem>
-      <Box mt="md">
-        <ConfirmButton size="md" variant="primary">确认</ConfirmButton>
-      </Box>
-    </Box>
-  </ModalPage>
-);
+export default function ResetPasswordPage() {
+  const {
+    validate, handleChange, values, touched, toggleTouched,
+  } = useForm(config);
+
+  const [submitting, setSubmitting] = useState(false);
+
+  const { query: { token }, push } = useRouter();
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
 
-export default ResetPasswordPage;
+    toggleTouched(true);
+
+    if (validate()) {
+      setSubmitting(true);
+
+      try {
+        await resetPassword(values.password, token);
+        push('/');
```

Comment:
这里用 `import Router, { useRouter } from 'next/router'`
```suggestion
        Router.push('/');
```

Reply:
这种写法不工作  报错Status Code: 400 Bad Request